plugins/utils.go

```diff
@@ -0,0 +1,26 @@
+package plugins
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+// LoopExecConfig is configuration used to register the LOOP and generate an exec
+type LoopExecConfig struct {
+	Id  string // unique string used by the node to track the LOOP. typically supplied by the loop logger name
```

Comment:
`ID`? or what about aligning withe `Name`?
```suggestion
	ID  string // unique string used by the node to track the LOOP. typically supplied by the loop logger name
```